### ISSUE TITLE
smart_text import error

### ISSUE THREAD

**Issue by chryzcode** [roverdotcom/django-inlinecss#62]
In the file [django_inlinecss/templatetags/inlinecss.py](https://github.com/roverdotcom/django-inlinecss/blob/master/django_inlinecss/templatetags/inlinecss.py), I've been getting smart_text error which is as a result of line 7 code `from django.utils.encoding import smart_text`.

**Comment by residentcode** [comment]
Yes in django 4 changed to smart_str https://docs.djangoproject.com/en/4.1/_modules/django/utils/encoding/
I have to change it manually .
looks django-inlinecss is discontinued
